CanvasPlusPlayground/PeopleManager.swift

```diff
@@ -111,7 +114,11 @@ class PeopleManager {
                         return
                     }
                     
-                    guard let enrollments: [Enrollment] = try? await CanvasService.shared.syncWithAPI(request) else {
+                    guard let enrollments: [Enrollment] = try? await CanvasService.shared.loadAndSync(request, onCacheReceive: { cached in
+                        guard let cached else { return }
+
+                        processEnrollments(cached)
```

Comment:
would this result in duplicates in `commonCourses` since we call processEnrollments twice in some cases